src/TableDetails.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const TableDetails = props => {
+  const [isSelected, setIsSelected] = useState([]);
```

Comment:
This way works, but there is an easier (cleaner) way that doesn't involve handling ids.

You can extract each table row into a separate component, then add state to each row component to keep track of whether it's selected or not (it would just be a boolean). Finally you need to make className dependent on the value in the state.

Let me know if you want to give it a try and need help with this.